python/pyspark/mllib/classification.py

```diff
@@ -428,9 +428,10 @@ def save(self, sc, path):
     def load(cls, sc, path):
         java_model = sc._jvm.org.apache.spark.mllib.classification.NaiveBayesModel.load(
             sc._jsc.sc(), path)
-        py_labels = _java2py(sc, java_model.labels())
-        py_pi = _java2py(sc, java_model.pi())
-        py_theta = _java2py(sc, java_model.theta())
+        # Can not unpickle array.array from Pyrolite in Python3 with "bytes"
```

Comment:
I had not figured out a way to make Pyrolite compatible both Python 2 and 3.

Another option is to use "latin1" as encoding, then the serialization of Vector will slow, because . the bytes will be converted into unicode, then converted into bytes again.